samples/bot-all-cards/csharp/README.md

```diff
@@ -0,0 +1,151 @@
+---
+page_type: sample
+description: Type of Cards.
```

Comment:
Check mohammed's PR and update this readme accordingly.

Reply:
Fixed